### DISCUSSION TITLE
Add task with priority

### DISCUSSION THREAD

**Discussion by hirschenberger** [VladimirMarkelov/ttdl#102] [Q&A] [answered]
Maybe I oversee something, but how can I add a task and assign a prio in one command?

```
ttdl "Buy milk pri:a" 
```
does not work.

**Comment by VladimirMarkelov** [comment, marked as answer]
You can prepend the required priority to the task's text. Something like this:

```
$ ttdl add "(A) high priority"
Added todo:
 # P D Created    Due Subject
------------------------------
15 A   2024-10-23     high priority
```

**Reply by hirschenberger** [reply]
Ah ok, make sense. Maybe you could add an example to the docs, it's not really shown somewhere.
